src/main/java/ch/njol/skript/expressions/ExprBlockSound.java

```diff
@@ -96,7 +95,7 @@ public boolean init(Expression<?>[] exprs, int matchedPattern, Kleenean isDelaye
 			.map(this::convertAndGetSound)
 			.filter(Objects::nonNull)
 			.distinct()
-			.map(Sound::name)
+			.map(sound ->SoundUtils.getKey(sound).getKey())
```

Comment:
Nitpick:
```suggestion
			.map(sound -> SoundUtils.getKey(sound).getKey())
```
do I care about this? no
do I want another fun commit name? yes

Reply:
ya bastard